vocabularies/voc4cat/0007015.ttl

```diff
@@ -0,0 +1,14 @@
+PREFIX dcterms: <http://purl.org/dc/terms/>
+PREFIX skos: <http://www.w3.org/2004/02/skos/core#>
+PREFIX xsd: <http://www.w3.org/2001/XMLSchema#>
+
+<https://w3id.org/nfdi4cat/voc4cat_0007015>
+    a skos:Concept ;
+    dcterms:identifier "voc4cat_0007015"^^xsd:token ;
+    skos:broader <https://w3id.org/nfdi4cat/voc4cat_0007003> ;
+    skos:definition "The physical shape of the catalyst sample, e.g. powder, sievefraction, thin film,…"@en ;
```

Comment:
- We can add as narrower concept thin film: https://w3id.org/nfdi4cat/voc4cat_0000019. The definition mentions a photocatalyst but we can change it to just catalyst.
-

Reply:
I removed this entry and added "catalyst form" as alternate label to "sample form"